recipes/paratext/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: paratext
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/wiseio/paratext.git
+  git_rev: 2f2824eb8c8f1d060c1105531bc7ab9c44b55246
```

Comment:
Done. They maintainer is also going to make a release so I will update to that commit/tag tarball when its done.